examples/custom_messages.py

```diff
@@ -0,0 +1,37 @@
+from locust import HttpUser, task, events, between
+from locust.runners import MasterRunner, WorkerRunner
+
+# Fired when the worker recieves a message of type 'test_users'
+def setup_test_users(environment, msg, **kwargs):
+    for user in msg.data:
+        print(f"User {user['name']} recieved")
+    environment.runner.send_message('acknowledge_users', f"Thanks for the {len(msg.data)} users!")
+
+# Fired when the master recieves a message of type 'acknowledge_users'
+def on_acknowledge(msg, **kwargs):
+    print(msg.data)
+
+@events.init.add_listener
+def on_locust_init(environment, **_kwargs):
+    if isinstance(environment.runner, WorkerRunner):
+        environment.runner.register_message('test_users', setup_test_users)
+    elif isinstance(environment.runner, MasterRunner):
+        environment.runner.register_message('acknowledge_users', on_acknowledge)
+
+@events.test_start.add_listener
+def on_test_start(environment, **_kwargs):
+    if isinstance(environment.runner, MasterRunner):
+        users = [
+            {"name": "User1"},
+            {"name": "User2"},
+            {"name": "User3"},
+        ]
+        environment.runner.send_message('test_users', users)  
```

Comment:
Does it make sense for the master to be able to send a message to a specific worker? Sending to all workers is also useful, but sending to a specific one is probably even more so.

Reply:
Can you give an example of a use-case where the targeting would be beneficial? In my opinion, since the Master node has so little information with which to target the nodes anyways, it'd be easier to either only register the events you'd want to target on the nodes you're interested in, or include the targeting criteria in the message data somehow, rather than trying to figure out from the master which node you'd want to receive the message.

Reply:
Actually after updating the `custom_messages` example I see the usefulness. I'll add that in as well. 👍